target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8065-rt4230w-rev6.dts

```diff
@@ -0,0 +1,373 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq8065.dtsi"
+#include <dt-bindings/input/input.h>
```

Comment:
please just drop the 4.19 files entirely to make this easier to review.

Reply:
I only have the 5.4 files now